docs-src/go/how-to-handle-an-update-in-go.md

```diff
@@ -0,0 +1,34 @@
+---
+id: how-to-handle-an-update-in-go
+title: How to handle an Update in Go
+sidebar_label: Handle Update
+description: Use the SetUpateHandler API from the go.temporal.io/sdk/workflow package to register an Update Handler for a given name.
+---
+
+Register an Update handler for a given name using the [SetUpdateHandler](https://pkg.go.dev/go.temporal.io/sdk/workflow#SetUpdateHandler) API from the `go.temporal.io/sdk/workflow` package.
+The handler function can accept multiple serializable input parameters, but we recommend using only a single parameter.
+This practice enables you to add fields in future versions while maintaining backward compatibility.
+You can optionally include a `workflow.Context` parameter in the first position of the function.
+The function can return either a serializable value with an error or just an error.
```

Comment:
Can you add something along the lines of "The handling of a panic in an Update handler function is governed by the workflow's WorkflowPanicPolicy configuration"